programs/clearing_house/src/controller/amm.rs

```diff
@@ -333,9 +333,15 @@ pub fn update_pool_balances(
 
     let mut fraction_for_amm = 100;
 
-    if market.amm.total_fee_minus_distributions <= amm_fee_pool_token_amount {
+    let amm_target_fee_pool_token_amount = market
+        .amm
+        .total_fee_minus_distributions
+        .checked_sub(cast_to_i128(market.amm.total_fee_withdrawn)?)
+        .ok_or_else(math_error!())?;
```

Comment:
maybe we eventually track this on state since its used in multiple places? total_fee_minus_distributions_and_withdraws

Reply:
useful to see the cumulative breakdown, subtracts are cheap relative to add/sub